SDK/AppCenterCrashes/Microsoft.AppCenter.Crashes.WindowsDesktop/Utils/StackTraceHelper.cs

```diff
@@ -0,0 +1,78 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Linq;
+
+namespace Microsoft.AppCenter.Crashes.Utils
+{
+    public class StackTraceHelper
+    {
+        // Exceptions don't always have complete stack traces, so they must be augmented.
+        public static string GenerateFullStackTrace(Exception e)
```

Comment:
I'm a bit confused as to why this is duplicated here from the one in Crashes.cs.  Does it have windows desktop-specific code?  If so, what code is not shared?